src/network/netdev/tunnel.c

```diff
@@ -909,7 +909,8 @@ int config_parse_ipv6_flowlabel(
                 void *userdata) {
 
         Tunnel *t = ASSERT_PTR(userdata);
-        int k, r;
+        unsigned k;
```

Comment:
`unsigned` -> `uint32_t` should be better, as `k` will be passed to `htobe32()`.